northpoint-development/Vector-Tiles-Google-Maps
# BMAPS-1691 Converted Mercator.js to ES6 and added JSDocs documentation

## Issue Link

[BMAPS-1691](https://northpoint-development.atlassian.net/browse/BMAPS-1691)

## Description

- Converted `Mercators.js` to ES6 syntax
- Removed `Mercator` folder and moved file to same level as `drawing.js` and `geometry.js`
- Adjusted imports in `MVTLayer.js`, `MVTSource.js`, and `Mercator.spec.js` to reflect new syntax style
- Added PR template 

**NOTE:** The tests in `Mercator.spec.js` already test each of the functions and I wasn't sure what to add here, so I have opted to not add any new tests. But I am open to any suggestions if we feel these changes warrant more tests!

## QA Steps
- Pull down this branch and run `npm install` if necessary
- Run `npm run dev` and open the page hosted by Vite
- Check each example and ensure they work

## Checklist
 
ALWAYS
- [x] My pull request title matches the following format: [project acronym]-### - [concise title]
      (Ex. 'BMAPS-123 Updated Pull Request Template')
- [x] I added/updated tests for the changes I made. (Or I have explained why I have not)
- [ ] I have run `npm audit fix` (Maybe multiple times) until all vulnerabilities that can be automatically fixed are fixed. If there are vulnerabilties that will need to be fixed manually, I have created a ticket to do so.

ALMOST ALWAYS
- [ ] I added or updated documentation/README

SOMETIMES
- [ ] (If this PR ends in a 0) I have scheduled a team code review

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

[BMAPS-1691]: https://northpoint-development.atlassian.net/browse/BMAPS-1691?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
